src/core/shiftManagement.ts

```diff
@@ -55,3 +62,47 @@ export function applyToShift(
     },
   });
 }
+
+export function rateWorker(
+  workerUuid: Worker["uuid"],
+  shiftUuid: Shift["uuid"],
+  rating: number,
+): Promise<ShiftAssignment> {
+  return prisma.shiftAssignment.update({
```

Comment:
The rateWorker function should include validation to ensure rating is between 1-5.